Patterns/Italian/Italian-Numbers.yaml

```diff
@@ -409,47 +419,75 @@ RoundNumberMap: !dictionary
     milioni: 1000000
     miliardo: 1000000000
     miliardi: 1000000000
-    trilione: 1000000000000
-    trilioni: 1000000000000
+    bilione: 1000000000000
+    bilioni: 1000000000000
+    trilione: 1000000000000000000
+    trilioni: 1000000000000000000
     centinaio: 100
     centinai: 100
     centinaie: 100
     millesimo: 1000
     milionesimo: 1000000
     miliardersimo: 1000000000
-    trilioneisimo: 1000000000000
+    bilionesimo: 1000000000000
+    trilionesimo: 1000000000000000000
     millesima: 1000
     milionesima: 1000000
     miliardersima: 1000000000
-    trilioneisima: 1000000000000
+    bilionesima: 1000000000000
+    trilionesima: 1000000000000000000
     millesimi: 1000
     milionesimi: 1000000
     miliardersimi: 1000000000
-    trilioneisimi: 1000000000000
+    bilionesimi: 1000000000000
+    trilionesimi: 1000000000000000000
     millesime: 1000
     milionesime: 1000000
     miliardersime: 1000000000
-    trilioneisime: 1000000000000
+    bilionesime: 1000000000000
+    trilionesime: 1000000000000000000
     centinaia: 100
     migliaia: 1000
     milionata: 1000000
     miliardata: 1000000000
-    trilionata: 1000000000000
+    bilionata: 1000000000000
+    trilionata: 1000000000000000000
     dozzina: 12
     dozzine: 12
     k: 1000
     m: 1000000
     g: 1000000000
     b: 1000000000
     t: 1000000000000
-RelativeReferenceOffsetMap: !dictionary
-  types: [ string, string ]
-# TODO: modify below regex according to the counterpart in English
-  entries:
-    '': ''
-RelativeReferenceRelativeToMap: !dictionary
+RelativeReferenceMap: !dictionary
   types: [ string, string ]
-# TODO: modify below regex according to the counterpart in English
   entries:
-    '': ''
+    ultimo: N
```

Comment:
The behaviour for ordinal.relative has changed in /master. Can you please sync with the latest behaviour in English?